engine/table/src/main/java/io/deephaven/engine/table/impl/by/AggregationContext.java

```diff
@@ -51,11 +51,6 @@
      */
     private final boolean requiresRunFinds;
 
-    /**
```

Comment:
Good question. We discussed in December, and you've likely forgotten due to the time gap.

`KeyOnlyAggregationFactory` thought it was telling the helper to not add back modifies, but it wasn't. It uses this constructor, which ignores `addedBackModified`:
```
    AggregationContext(IterativeChunkedAggregationOperator[] operators, String[][] inputNames,
            ChunkSource.WithPrev<Values>[] inputColumns, boolean addedBackModified) {
        this(operators, inputNames, inputColumns, null, true);
    }
```

We concluded that `addedBackModified` was in fact nonsense and could not change the `modified` set if the operators are correctly advising the helper of modifications. You'll note that I've deleted the lines for that insert from the helper:
```
                    if (ac.addedBackModified()) {
                        downstream.modified().writableCast().insert(addedBack);
                    }
```                    

So `selectDistinct` won't produce any modifies since it has no operator to tag rows modified.